reviews/serializers.py

```diff
@@ -60,5 +83,44 @@ class ReviewLogSerializer(JSONAPISerializer):
     def get_log_url(self, obj):
         return absolute_reverse('reviews:review_log-detail', kwargs={'log_id': obj._id, 'version': self.context['request'].parser_context['kwargs']['version']})
 
+    def create(self, validated_data):
+        action = validated_data.pop('action')
+        user = validated_data.pop('user')
+        reviewable = validated_data.pop('reviewable')
+        comment = validated_data.pop('comment', '')
+
+        try:
+            if action == Actions.ACCEPT.value:
```

Comment:
I feel like there should be a better way to do this. This is the most straightforward way and it problem wont change soon so it's probably fine?